Microsoft.Windows.Compatibility depends on non-existent package

In p5, I see this issue in package validation: https://dev.azure.com/dnceng/internal/_build/results?buildId=1166651&view=logs&j=38f37d44-1649-5acd-0b92-165e751e1470&t=e3a67328-30a8-5faa-9e47-9d1f680342f0&l=752

```
##[error]Microsoft.Windows.Compatibility.6.0.0-preview.5.21301.5.nupkg's dependency 'System.Security.Cryptography.Cng' upon versions: [6.0.0-preview.5.21301.5, ) not found locally nor in nuget.org
```

The System.Security.Cryptography.Cng package does not appear to have been produced since p4.

--- a/src/libraries/pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj
+++ b/src/libraries/pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj
@@ -36,7 +36,6 @@
     <PrereleaseLibraryPackage Include="System.Management" />
     <PrereleaseLibraryPackage Include="System.Runtime.Caching" />
     <PrereleaseLibraryPackage Include="System.Security.AccessControl" />
-    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Cng" />
     <PrereleaseLibraryPackage Include="System.Security.Cryptography.Pkcs" />
     <PrereleaseLibraryPackage Include="System.Security.Cryptography.ProtectedData" />
     <PrereleaseLibraryPackage Include="System.Security.Cryptography.Xml" />
@@ -52,21 +51,12 @@
     <NS21PrereleaseLibraryPackage Include="System.Reflection.Context" />
 
     <!-- Packages we don't build in main anymore -->
-    <LibraryPackage Include="System.Data.DataSetExtensions">
-      <Version>4.5.0</Version>
-    </LibraryPackage>
-    <LibraryPackage Include="System.Data.SqlClient">
-      <Version>4.8.1</Version>
-    </LibraryPackage>
-    <LibraryPackage Include="System.Reflection.Emit">
-      <Version>4.7.0</Version>
-    </LibraryPackage>
-    <LibraryPackage Include="System.Reflection.Emit.ILGeneration">
-      <Version>4.7.0</Version>
-    </LibraryPackage>
-    <LibraryPackage Include="System.Reflection.Emit.Lightweight">
-      <Version>4.7.0</Version>
-    </LibraryPackage>
+    <LibraryPackage Include="System.Data.DataSetExtensions" Version="4.5.0" />
+    <LibraryPackage Include="System.Data.SqlClient" Version="4.8.2" />
+    <LibraryPackage Include="System.Reflection.Emit" Version="4.7.0" />
+    <LibraryPackage Include="System.Reflection.Emit.ILGeneration" Version="4.7.0" />
+    <LibraryPackage Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
+    <LibraryPackage Include="System.Security.Cryptography.Cng" Version="5.0.0" />
 
     <!-- Service model packages -->
     <ExternalLibraryPackage Include="System.ServiceModel.Primitives">

